### DISCUSSION TITLE
容器无法运行，这是bug还是我没配置好？

### DISCUSSION THREAD

**Discussion by reedlxm** [bfenetworks/bfe#1090] [General]
docker run -p 8080:8080 -p 8443:8443 -p 8421:8421 -v /bfe/log:/bfe/log -v /bfe/conf:/bfe/conf bfenetworks/bfe
2022/12/01 10:05:19 maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined

**Comment by iyangsj** [comment]
输出日志是告警信息，不影响启动运行。 

按这里的方式，确认是否成功运行：https://github.com/bfenetworks/bfe/blob/develop/docs/zh_cn/installation/install_using_docker.md

如果没有成功运行，可以检查下bfe日志中具体输出的错误信息

**Comment by reedlxm** [comment]
[EROR] [1] (main.main:99) main(): in BfeConfigLoad():warning:
can't store data at section "Server", variable "MonitorEnabled"
log里输出的错误日志是这样。配置是默认配置，MonitorEnabled = true

**Reply by reedlxm** [reply]
不挂载目录的情况可以运行起来。挂载目录就不行。但是log可以写进挂载的目录里面，应该不是权限的问题吧

**Reply by reedlxm** [reply]
应该就是配置文件不对，不挂载配置目录容器就可以起来。挂载就是错误，找不到配置文件。我再看看吧。
